apps/src/code-studio/pd/workshops/components/UserPassport.tsx

```diff
@@ -78,57 +81,73 @@ const UserPassport: React.FunctionComponent<{
 
   return (
     <div className={classNames(style.userInfoContainer, className)}>
-      <span className={style.userInfoHeader}>
+      <span
```

Comment:
If these changes are specific to the workshop marketing page, wondering if it's worth updating `UserPassport` here since if `isUserEnrolled` is true then we don't show most of the UserPassport or use any of the logic (i.e. there's no edit buttoin). I feel like it may be easier to just have it conditionally render either the user icon (with the enrolled tag) or the entire UserPassport.

Reply:
That makes sense! I think it wouldn't ever show an enrolled-version of the Join page though since there's a separate "You are already enrolled in this workshop" view if an enrolled user tries to access the join page.